user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Low_Code_Apps/Low-Code_Apps_configuration/LowCodeApps_organizing_landing_page.md

```diff
@@ -31,3 +31,6 @@ To do so:
          "AllowEdit":[]
       }
    }
+
+> [!NOTE]
+> From DataMiner 10.2.9/10.3.0 onwards, these different sections will also be displayed in the apps pane in DataMiner Cube.<!-- RN 33944 -->
```

Comment:
Not sure why, but it's showing a red background color. Could it be due to indentation?

Reply:
I'm not sure why either – the syntax looks fine in VS Code, and if I generate a test build, everything looks OK.